.pre-commit-config.yaml

```diff
@@ -9,11 +9,11 @@ repos:
         rev: 22.10.0
         hooks:
         -   id: black
-    -   repo: https://github.com/timothycrosley/isort
```

Comment:
Thanks for fixing this! Mind moving it to a separate PR?

Reply:
Sure! it just was in my way when trying to commit my other changes :)

Reply:
Done in #679